clamd/clamav-daemon.service.in

```diff
@@ -12,6 +12,30 @@ ExecStart=@prefix@/sbin/clamd --foreground=true
 ExecReload=/bin/kill -USR2 $MAINPID
 TimeoutStartSec=420
 
+##
+## Security Hardening Options
+##
+
+# Remove `ProtectSystem`, `ProtectHome`, and `ReadWritePaths`
+# if you want ClamAV to be able to remove infected files.
+ProtectSystem=strict
+ProtectHome=read-only
+ReadWritePaths=/var/log/
```

Comment:
It occurs to me that we would need a read-write path for  for the socket file.  
```suggestion
ReadWritePaths=/var/log/ /var/run/
```
Granted `/var/run` isn't the only place one my choose to put the socket file, but it is the most likely.

Reply:
I am also adding `/run`. We should be able to use a `RuntimeDirectory=` directive and then use the `$RUNTIME_DIRECTORY` internal variable to add to `ReadWritePaths=`